api/src/models/ImageFile.ts

```diff
@@ -74,29 +80,33 @@ class ImageFile {
         let fs = require("fs");
         let png = this.derivativePath('ocr/pngs', 'png');
         let { exec } = require("child_process");
-        if (fs.existsSync(png)) {
-            let tc_cmd = "#{config['textcleaner_path']} #{config['textcleaner_switches']} #{derivative('LARGE')} #{png}"
+        if (!fs.existsSync(png)) {
+            let tc_cmd = this.config().textcleanerPath() + this.config().textcleanerSwitches() + this.derivative('LARGE') + png;
```

Comment:
You're going to need to add `+ " "` in between all of these things so the parameters don't run together.

Reply:
I also think you might need to `await` the call to `this.derivative()` because it's an `async` call...